site/i18n/en/docusaurus-plugin-content-docs/current/project/intro.md

```diff
@@ -0,0 +1,44 @@
+# Noslate Project Overview
+
+Noslate is an exploration of the evolution of the Node.js/V8 technology stack in the cloud-native era, aiming to improve the scheduling performance and diagnosing black-box problems of JavaScript in cloud-native scenarios. The goal is to allow JavaScript Workload to have higher scheduling flexibility in the cloud-native era. At present, the relevant software has been applied in some core scenarios of online services links. This project mainly consists of three sub-projects: Workers, Debugger, and Node.js Distribution.
```

Comment:
At present, the relevant software has been used in some enterprise online services.